browsers-internal/releases/node-18/node18.17.1-chrome128-ff130/global-profile.sh

```diff
@@ -0,0 +1,11 @@
+if [[ "$(uname -a)" = *"arm"* || "$(uname -a)" = *"aarch64"* ]]; then
+    printf "\e[31m" # red
+    echo "Warning: You are using the beta Arm build of a cypress/browsers or cypress/included image."
+    echo
+    echo "On Arm, non-Electron browsers are not available, because browser vendors are not yet building for Linux arm64."
+    echo "You must use the built-in Electron browser (--browser electron) to run Cypress or find and install unofficial Arm binary builds."
+    echo
+    echo "More details and links to upstream issues for Chrome, Firefox, and Edge can be found at Cypress's issue tracker:"
```

Comment:
Same comment as above

https://github.com/cypress-io/cypress-docker-images/blob/master/README.md#browsers

Reply:
misunderstood. For some reason I thought a few of the arm browsers were available 🥴 . I have the global-profile updated in 42eb576

Reply:
Some of the `arm64` browsers **are** available as Debian packages. You just can't select the version you want, so they're not suitable for selection and installation through the `.env` / `factory` process.

You can however install with `apt-get install`